### ISSUE TITLE
/usr/lib/libmesos-1.2.0.so: libcurl-nss.so.4: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by albundy83** [mesosphere/mesos-deb-packaging#102]
Hello,
I'm under debian 8 and I have just upgraded to mesos 1.2.
Then mesos failed to start.
In /var/log/syslog I got the following error:
```bash
java.lang.UnsatisfiedLinkError: /usr/lib/libmesos-1.2.0.so: libcurl-nss.so.4: cannot open shared object file: No such file or directory
```
Installing libcurl4-nss-dev fix the problem.
It's a problem that seems already covered by https://github.com/mesosphere/mesos-deb-packaging/pull/48

**Comment by albundy83** [comment]
I have created a fresh pull request (https://github.com/mesosphere/mesos-deb-packaging/pull/103) as the previous one got some conflicts.

**Comment by chriskuehl** [comment]
I don't believe you actually need `libcurl4-nss-dev`. Instead I think you need `libcurl3-nss`.

Despite the confusing name, this package is what provides `libcurl-nss.so.4`:

```bash
$ dpkg -L libcurl3-nss | grep libcurl-nss.so.4
/usr/lib/x86_64-linux-gnu/libcurl-nss.so.4.3.0
/usr/lib/x86_64-linux-gnu/libcurl-nss.so.4
```

The `-dev` package you identified also works, but that's just because it happens to depend on `libcurl3-nss`. I don't believe the development headers are actually necessary.

**Comment by albundy83** [comment]
Yes, you are right, that's much better.
I have updated pull request according https://github.com/mesosphere/mesos-deb-packaging/pull/104

**Comment by timcharper** [comment]
FYI I've talked with @kapilsla; the build script is being updated to not depend on libcurl-nss for debian packages anymore. That was an error. The upcoming 1.7.0 Mesos package should depend on libcurl-openssl and have the proper dependencies.
